frontend/src/components/datasets/ExpandableTableDatasetComments.tsx

```diff
@@ -9,7 +9,7 @@ import { Dataset } from "../../models/frontend_models";
 export default function ExpandableTableDatasetComments(): NonNullable<TableProps<any>['expandable']> {
   return (
     {
-      columnTitle: <div>Comments</div>,
+      columnTitle: <CommentOutlined />,
```

Comment:
Maybe add a hovering tooltip "Validation comments" to the icon ?

Reply:
tooltip added